smartcontract/service/native/auth/auth.go

```diff
@@ -307,29 +336,31 @@ func getLevel(native *native.NativeService, contractAddr, ontID, role []byte) (u
  * then make changes to storage as follows:
  */
 func delegate(native *native.NativeService, contractAddr []byte, from []byte, to []byte,
-	role []byte, period uint32, level uint8, keyNo uint64) ([]byte, error) {
+	role []byte, period uint32, level uint8, keyNo uint64) (bool, error) {
```

Comment:
the type of param `from` `to` are Address?

Reply:
No, they are ONT ID